src/project/PackageFactory.js

```diff
@@ -173,41 +173,13 @@ define(function (require, exports, module) {
     }
 
     /**
-     * Check if the given dependency name is defined in bower json dependencies.
-     * @param {string} name
-     * @param {object} bowerJsonDependencies
-     * @return {string}
-     * @private
-     */
-    function _getBowerJsonVersion(name, bowerJsonDependencies) {
-        var version = null;
-
-        if (!bowerJsonDependencies) {
-            return version;
-        }
-
-        if (bowerJsonDependencies.dependencies &&
-                bowerJsonDependencies.dependencies[name]) {
-
-            version = bowerJsonDependencies.dependencies[name];
-        } else if (bowerJsonDependencies.devDependencies &&
-                bowerJsonDependencies.devDependencies[name]) {
-
-            version = bowerJsonDependencies.devDependencies[name];
-        }
-
-        return version;
-    }
-
-    /**
      * Create package from raw data.
      * @param {string} name
      * @param {object} data
-     * @param {object=} bowerJsonDeps
      * @return {Package} pkg
      * @private
      */
-    function _packageFromRawData(name, data, bowerJsonDeps) {
+    function _packageFromRawData(name, data) {
```

Comment:
This does not need to be addressed right now, but probably it is worth re-thinking some function naming. Names such as `_packageFromRawData` and `_packageFromRawDataUsingBowerJson` are little too verbose and not intuitive (e.g. what's "raw data" is not immediately clear).